README.md

```diff
@@ -141,15 +141,15 @@ git clone https://github.com/kubernetes-client/gen
 
 ```bash
 # Where REPO_DIR points to the root of the csharp repository
-cd 
+cd
 ${GEN_DIR}/openapi/csharp.sh ${REPO_DIR}/src/KubernetesClient ${REPO_DIR}/csharp.settings
 ```
 
-# Version Compatibility 
+# Version Compatibility
 
 | SDK Version | Kubernetes Version | .NET Targeting                        |
 |-------------|--------------------|---------------------------------------|
-| 7.0         | 1.23               | netstandard2.1;net5;net6              |
+| 7.0         | 1.23               | netstandard2.1;net5.0;net6.0          |
 | 6.0         | 1.22               | netstandard2.1;net5                   |
```

Comment:
seems still net5 here

BTW, any doc link for `net5.0` is recommend way? just curious.

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/frameworks